pkg/arvo/sys/arvo.hoon

```diff
@@ -784,26 +869,21 @@
   ::
   ++  va
     =>  |%
-        ::  XX move next to +slap
-        ::
-        ++  swat
-          |=  [vax=vase gen=hoon]
-          ^-  (trap vase)
-          =/  gun  (~(mint ut p.vax) %noun gen)
-          |.([p.gun .*(q.vax q.gun)])
+        +$  vane-sample  [now=@da eny=@uvJ rof=rook]
```

Comment:
Curious what the reason is for passing `our` in through a top-level gate, as opposed to not having a top-level gate and just keeping it in this sample.

Reply:
The main motivation is to only pass values that change on each invocation.